pkg/test/resourcefixture/testdata/basic/bigqueryconnection/v1alpha1/bigqueryconnectionconnection/create.yaml

```diff
@@ -17,8 +17,10 @@ kind: BigQueryConnectionConnection
 metadata:
   name: bigqueryconnectionconnection-${uniqueId}
 spec:
+  friendlyName: "cloud-resource-connection"
+  description: "test connection for cloud resource"
   location: us-central1
   projectRef:
     external: ${projectId}
   cloudResource:
-    serviceAccountId: ""
\ No newline at end of file
+    serviceAccountId: ""
```

Comment:
nit: I think we want an extra line in the end

Reply:
There *is* an extra line at the end of the file. If there is not, it'll show the red sign. E.g. https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1981/files#diff-ef5a4fb16c799f5e257555474294ae1b67dd2c733712e98c3b6db2a2e48aaf85